systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -219,4 +223,31 @@ public static void waitForTopicWithPrefixDeletion(String namespaceName, String t
                 }
             });
     }
+
+    /**
+     * Verifies that {@code absentTopicName} topic remains absent in {@code clusterName} Kafka cluster residing in {@code namespaceName},
+     * by querying the cluster using kafka scripts from {@code queryingPodName} Pod.
+     * @param namespaceName Namespace name
+     * @param queryingPodName  the name of the pod to query Kafka topics from
+     * @param clusterName name of Kafka cluster
+     * @param absentTopicName name of Kafka topic which should not be created.
+     * @throws AssertionError in case topic is created
+     */
+    public static void verifyUnchangedTopicAbsence(String namespaceName, String queryingPodName, String clusterName, String absentTopicName) {
+
+        long poolInterval = Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS; // repeat every second
+        long duration = Constants.RECONCILIATION_INTERVAL; // repeat for ~30sec
```

Comment:
I think you can use the constants directly

Reply:
default is 120 seconds (AFAIK) but i explicitly set 10 seconds in  kafka spec for TO reconciliation, so I can pass it (10sec pool interval) as well.